presto-native-execution/presto_cpp/main/PeriodicTaskManager.h

```diff
@@ -68,6 +68,14 @@ class PeriodicTaskManager {
         func, std::chrono::microseconds{periodMicros}, taskName);
   }
 
+  /// Add a task to run once.
+  template <typename TFunc>
+  void
+  addTaskOnce(TFunc&& func, size_t periodMicros, const std::string& taskName) {
```

Comment:
Can you add a unit test with a test function to verify if it is only called once? Like verify by a counter incremented by the test function and wait for the test run a bit to verify the counter only incremented once? Thanks!